src/lib/Link.svelte

```diff
@@ -53,7 +53,7 @@
   // this will enable `<Link on:click={...} />` calls
   function onClick(e) {
     // user used a keyboard shortcut to force open link in a new tab
-    if (e.target.tagName === 'A' && (e.metaKey || e.ctrlKey || e.button !== 0)) {
+    if (e.metaKey || e.ctrlKey || e.button !== 0) {
```

Comment:
I'm still aware if this affecting buttons too, we must check the actual target node. But instead of iterating `parentNode` we could pass an additional argument to differentiate between anchors and buttons, e.g.

```
function onClick(e, isAnchor) {
  if (isAnchor && (...)) return;
  ...
```

and then, on the template invoking the method `on:click={e => onClick(e, true)}` directly?

Reply:
AHH! I see your concern.. I have an idea, lemme update some code.

Reply:
Since we're already in control of our two Dom elements, rather than going crazy with checking for things, I just created a small handler for the anchor tag, and if it's a an open in tab, bounce, if not, delegate to the other handler.

Reply:
thoughts?

Reply:
Yeah, it would be fine, just move the new function after or eslint will complain ;-)

Reply:
yea my bad on that one.. I'm a bit to use to my own build scripts and forgot to run your test.